docs/en/deploy/Running-Alluxio-On-a-Cluster.md

```diff
@@ -36,6 +36,7 @@ To deploy Alluxio in production, we highly recommend running Alluxio masters in
   For basic functionality, make sure RPC port (default:19998) is open on all nodes.
 * Allow `sudo` privilege for the OS user that Alluxio will be running as.
   This is only needed if you expect Alluxio to mount a RAMFS on the workers automatically.
+* Enable sudo command can be used without inputting password.
```

Comment:
```suggestion
* Ensure that the user Alluxio will be running as can execute `sudo` without inputting a password.
```

Also can you please add a link to the section from the other doc where there are instructions on how to set `NOPASSWD` config for `sudo`?